src/accel/SharedParams.cc

```diff
@@ -320,6 +322,24 @@ SharedParams::SharedParams(SetupOptions const& options)
     // Translate supported particles
     particles_ = build_g4_particles(params_->particle(), params_->physics());
 
+    // Create bounding box from navigator geometry
+    bbox_ = [] {
+        G4VPhysicalVolume const* world = GeantImporter::get_world_volume();
+        CELER_ASSERT(world);
+        G4LogicalVolume const* world_lv = world->GetLogicalVolume();
+        CELER_ASSERT(world_lv);
+        G4VSolid const* solid = world_lv->GetSolid();
+        CELER_ASSERT(solid);
+        G4VisExtent const& extent = solid->GetExtent();
+
+        BBox result{{extent.GetXmin(), extent.GetYmin(), extent.GetZmin()},
+                    {extent.GetXmax(), extent.GetYmax(), extent.GetZmax()}};
+        CELER_VALIDATE(result,
+                       << "world bounding box {" << result.lower() << ", "
+                       << result.upper() << "} is invalid");
+        return result;
+    }();
```

Comment:
I was actually working on that in parallel but didn't add diagnostic 😅 

Can we retrieve the bounding box from `params.geant_geo_params()->bbox()` instead of constructing it in place. Also, doesn't it make more sense semantically for the `LocalTransporter` to hold the Bounding box(or keep a reference to `SharedParams`?

Reply:
The geo bbox could be slightly different: it is expanded by VecGeom and may have round off error. And yes the semantics make more sense to reference, but we can't store a safe reference to the shared params.

Reply:
That probably should be updated to match the realities of the Geant4 allocators...  we have to allow `LocalTransporter` to take a `SharedParams*` because we don't want to enforce users having shared pointers. (Indeed, our singletons and most of our existing implementations use `G4ThreadLocal`.)

Anyway, it probably doesn't matter too much about which bbox we use; I figured making this part of code more Geant4-y would be ok.